manifests/gcp_marketplace/chart/kubeflow-pipelines/values.yaml

```diff
@@ -17,6 +17,8 @@ images:
 
 gcpSecretName: "user-gcp-sa"
 serviceAccountCredential: ""
+gcsConfigName: "kfp-default-storage-config"
```

Comment:
is it OK to use a more general name so that in the future we can avoid introducing many small sized ConfigMap but put key-value pairs to a common ConfigMap? I'm also OK with current change, just want to help reduce number of ConfigMap.

Reply:
I think later on if we need to compose multiple storage config into one configmap we can rename this one.

Is there a concern regarding KFP upgrading story?

Reply:
rename configmap largely requires bump version. anyway, LGTM current change.